test/distributed/_tensor/test_dtensor_compile.py

```diff
@@ -161,22 +161,18 @@ def test_2d_fsdp_tp_compile(self):
             tp_model, process_group=fsdp_pg, device_id=self.rank, use_orig_params=True
         )
         out = eager_2d(inp)
-        # TODO: once aot autograd support is ready we can just use default backend
         tp_model2 = parallelize_module(
             model_copy, twod_mesh, PairwiseParallel(), tp_mesh_dim=1
         )
-        compiled_tp = torch.compile(tp_model2, backend="eager", fullgraph=True)
-
-        # TODO: now we first apply torch compile on tp model then use fsdp to wrap it, ideally
-        # we should apply torch.compile after fsdp wrap, but the current graph break approach
-        # have some issues with the tensor subclass compilation, need to dig into this later
-        compiled_2d = FSDP(
-            compiled_tp,
+        fsdp_2d = FSDP(
+            tp_model2,
             process_group=fsdp_pg,
             device_id=self.rank,
             use_orig_params=True,
         )
 
+        # TODO: once aot autograd support is ready we can just use default backend
+        compiled_2d = torch.compile(fsdp_2d, backend="eager")
```

Comment:
nit: we might also want to test inductor backend, as it does break sometimes more than eager, due to customized logic for some parts of aot-autograd

Reply:
aot autograd is currently broken until @bdhirsh aot dispatch subclass PR landed https://github.com/pytorch/pytorch/pull/104483, will switch this test to test both aot_eager or default backend once it's there :)